packages/webdriverio/src/utils.js

```diff
@@ -426,12 +426,13 @@ export async function getElementRect(scope) {
 
         // try set proper value
         Object.keys(defaults).forEach(key => {
-            if (rect[key] == null && typeof rectJs[key] === 'number') {
-                rect[key] = Math.floor(rectJs[key])
-            } else {
-                log.error('getElementRect', rect)
-                log.error('getBoundingClientRect', rectJs)
-                throw new Error('Failed to receive element rects via execute command')
+            if (rect[key] == null) {
```

Comment:
Let's do 
```js
if (rect[key] != null) {
    return
}
```
and save indentation.

Reply:
Yep, that's how I usually code but I didn't know whether you follow this approach. Changed it.